modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -137,9 +136,13 @@ export default class MeshLayer extends Layer {
       instanceRotations: {
         size: 3,
         accessor: ['getYaw', 'getPitch', 'getRoll'],
-        update: this.calculateInstanceRotations
+        defaultValue: [0, 0, 0]
```

Comment:
should we match the order of array elements in `accessor` with how they used in `calculateInstanceRotations` ?